doozer/doozerlib/cli/images_streams.py

```diff
@@ -796,6 +796,7 @@ def search_issues(query):
         fields = {
             'project': {'key': project},
             'issuetype': {'name': 'Bug'},
+            'labels': ['art:reconciliation', f'art:distgit:{distgit_key}'],
```

Comment:
I think `imagemeta` should have `get_component_name()`, which returns the brew component. Think there's some value in uniformity here.

Reply:
I made a wrong assumption that get_component_name retrieves the component in product.yaml page.

Updated